core/templates/app.constants.ajs.ts

```diff
@@ -18,12 +18,13 @@
 
 // TODO(#7092): Delete this file once migration is complete and these AngularJS
 // equivalents of the Angular constants are no longer needed.
-import { AppConstants } from 'app.constants';
-
 import constants from 'assets/constants';
+// // Import from assets/constants because we loop over all constants.
```

Comment:
I do not understand this comment.

Reply:
maybe we don't need this comment, I will delete it.

Reply:
done